Housing/README.md

```diff
@@ -1,18 +1,21 @@
 # Housing Assistance Program Imputation
 
-Housing assistance, is reported on a household, rather than
+Federal housing rental assistance is reported on a household, rather than
 individual level in the CPS. However, the actual benefit, and its corresponding outlay, are received at the family level.
 
 Target data for the imputation came from official [HUD data](https://www.huduser.gov/portal/datasets/assthsg.html),
 called "_Picture of Subsized Households_". To get 2014 data go to the bottom of the landing page and select year 2014,
 at the state summary level, with the summary of all programs and all variables. 
-In Fiscal Year 2014, around 4.5 million families claimed
-roughly 36 billion dollars in federal housing assistance benefits.
```

Comment:
Actually what I meant was just adding a sentence including two CPS aggregate numbers corresponding to 4.5 million and 36 billion. Could you add that? 

The other program details are great as well, so no need to remove what you just added.